src/Pessoa.php

```diff
@@ -681,4 +681,63 @@ public static function obterEndereco($codpes){
         }
         return false;
     }
+
+    /**
+     * Método para retornar o total de servidores ativos na unidade do gênero especificado
+     * @param Integer $codpes
+     * @return int|bool
+     */
+    public static function contarServidoresAtivosPorGenero($sexpes){
+        $query = " SELECT COUNT (DISTINCT LOCALIZAPESSOA.codpes) FROM LOCALIZAPESSOA 
+                    JOIN PESSOA ON PESSOA.codpes = LOCALIZAPESSOA.codpes 
+                    WHERE LOCALIZAPESSOA.tipvinext LIKE 'Servidor'
+                    AND LOCALIZAPESSOA.codundclg IN (getenv('REPLICADO_CODUNDCLG'))
+                    AND PESSOA.sexpes = :sexpes ";
+        $param = [
+            'sexpes' => $sexpes,
+        ];
+        $result = DB::fetch($query, $param);
+        if (!empty($result)) {
+            return $result;
+        }
+        return false;
+    }
+
+    /**
+     * Método para retornar o telefone principal e o celular da pessoa com número USP especificado
+     * @param Int
+     * @return String
+     */
+    public static function obterTelefones($codpes){
```

Comment:
Se for seguir o padrão o nome do método seria listarTelefoneFromCodpes e eu acho que deveria retornar todos os telefones se não deveria especificar no nome do método